doc/gamepad.md

```diff
@@ -1,3 +0,0 @@
-# Gamepad
-
-Flame has a separate plugin for gamepad support, please checkout the plugin [github project](https://github.com/fireslime/flame_gamepad) for more information.
```

Comment:
maybe not having a whole file dedicated to this, but we could at least add a section to the input.md file mentioning this? seems it's good to improve discoverability even though it's a standalone package